lib/imagemounter/index.js

```diff
@@ -0,0 +1,119 @@
+import { readFile, createReadStream } from 'fs';
+import { BaseServicePlist } from '../base-service';
+import { fs } from '@appium/support';
+import B from 'bluebird';
+import log from '../logger';
+const MOBILE_IMAGE_MOUNTER_SERVICE_NAME = 'com.apple.mobile.mobile_image_mounter';
+const { exists, lstat } = fs;
+class ImageMounter extends BaseServicePlist {
+
+    _checkReturnError(ret) {
+        if (ret.Error) {
+            throw new Error(ret.Error);
+        }
+    }
+
+    /**
+     * Lookup for mounted images.
+     * @param {string?} imageType Type of image, `Developer` by default.
```

Comment:
The assignment in the function signature _implies_ that `imageType` is optional, so you can just call it `string`:

```suggestion
     * @param {string} imageType Type of image, `Developer` by default.
```

`string?` in jsdoc means `string|null`, which is different than TS, where `?` _never_ means `null`.  This is confusing, and I'm recommending we don't use it (I'll see if I can come up with an eslint rule for this).

If you meant "`imageType` can be `null` or `string` or otherwise undefined", then use:

```suggestion
     * @param {string|null} imageType Type of image, `Developer` by default.
```

I don't think you meant this, though.